.gitignore

```diff
@@ -6,3 +6,14 @@ cmake-build-debug/
 .idea/
 
 /compile
+
+_binary2022_/
```

Comment:
You will remove them from .gitignore before the final merge, right?

Reply:
> You will remove them from .gitignore before the final merge, right?


ok I will

Reply:
> > are you 100% sure that it is necessary to turn the examples subfolder into a cmake project?
> 
> Its convenient!! what's wrong with having a project for the examples??

I think you placed your comment at the wrong conversation :)

Reply:
No actually there was no way to answer to your post regarding examples/CMakeLists.txt so I had to answer here!!
Why is that?

![image](https://user-images.githubusercontent.com/5367573/175380530-7c247ccb-d5ab-49d2-aafc-64faff85825e.png)